src/Appwrite/Utopia/Database/Validator/Spatial.php

```diff
@@ -32,10 +57,6 @@ public function __construct(string $spatialAttributeType)
      */
     public function isValid($value): bool
     {
-        if (!parent::isValid($value)) {
-            return false;
-        }
-        $value = \json_decode($value, true);
         $validator = new SpatialValidator($this->spatialAttributeType);
         return $validator->isValid($value);
```

Comment:
If we are not doing anything different here anymore, let's just use the DB validator directly